wp-includes/widgets/class-wp-widget-image.php

```diff
@@ -29,9 +29,16 @@ public function __construct() {
 		) );
 
 		$this->l10n = array_merge( $this->l10n, array(
-			'no_media_selected' => __( 'No image selected' ),
-			'edit_media' => __( 'Edit Image' ),
 			'change_media' => __( 'Change Image' ),
+			'edit_media' => __( 'Edit Image' ),
+			'missing_attachment' => sprintf(
+				/* translators: placeholder is URL to media library */
+				__( 'We can&#8217;t find that image. Check your <a href="%s">media library</a> and make sure it wasn&#8217;t deleted.' ),
+				esc_url( admin_url( 'upload.php' ) )
+			),
+			/* translators: %s is widget count */
+			'media_library_state' => _n_noop( 'Image Widget', 'Image Widget (%s)' ),
```

Comment:
I don't think that's a valid concern in our case. We're not saying `One  Widget`, `2  Widgets`, the label's singular/plural version is of no concern here really.

IMO we should rather skip the rule for that line than to introduce a language workaround (which in German at least reads quite awkwardly), what do you think?